Resources/Private/Language/de.locallang.xlf

```diff
@@ -591,6 +591,10 @@ Sächsische Landesbibliothek- Staats- und Universitätsbibliothek Dresden
 			<source>Type</source>
 			<target>Dokumenttyp</target>
 		</trans-unit>
+		<trans-unit id="search.form.placeholder">
+			<source>yyyy-mm-dd</source>
```

Comment:
This doesn't work.

1. We use little endian dates usually (DD/MM/YYYY or MM/DD/YYYY). See https://en.wikipedia.org/wiki/Date_format_by_country
2. What is the default dateform of the bootstrap datepicker? The jQuery-ui datepicker uses DD/MM/YYYY as default.The current datetimepicker returns DD.MM.YYYY. Don't know, where to configure this.
4. The AbstractSearchController expects "d.m.Y" (!)

Reply:
How to sort this out? Any thoughts, @ruegamer?

Reply:
You could detect the browser language with $_SERVER['HTTP_ACCEPT_LANGUAGE'] and write a second method convertFormDateEN for english in the AbstractSearchController.
This snippet from qucosa.js handles the dateformat of the datepicker:

var datepicker = function() {
    var language = jQuery('div.tx-dpf[data-language]').first().attr('data-language');
    if (!language) language = "en";
    jQuery('.datetimepicker').datetimepicker({
        useCurrent: false,
        format: 'DD.MM.YYYY',
        locale: language,
        keepInvalid: true
    });
}